src/Microsoft.Azure.WebJobs.Extensions.Kafka/Trigger/MultipleItemFunctionExecutor.cs

```diff
@@ -39,8 +43,19 @@ protected override async Task ReaderAsync(ChannelReader<IKafkaEventData[]> reade
                         {
                             TriggerValue = triggerInput,
                         };
-
-                        var functionResult = await this.ExecuteFunctionAsync(triggerData, cancellationToken);
+                        FunctionResult functionResult;
```

Comment:
This is a complete behavior of execution and adding activity & setting status so should be a separate method

Reply:
Added a new method for setting status